makefile

```diff
@@ -16,11 +18,17 @@ help:
 
 # Initialization ===============================================================
 copy-conf: ## Initialize the configuration files by copying the *''-dist" versions (does not override existing config)
-	-cp -n ./config/development-dist.js ./config/development.js
+	-cp -n ./config/${NODE_ENV}-dist.js ./config/${NODE_ENV}.js
+ifeq ($(NODE_ENV), development)
```

Comment:
and if it didn't work, travis won't pass because it'll miss the babel/webpack/selenium packages